ssh-key/Cargo.toml

```diff
@@ -80,6 +80,7 @@ p256 = ["dep:p256", "ecdsa"]
 p384 = ["dep:p384", "ecdsa"]
 p521 = ["dep:p521", "ecdsa"]
 rsa = ["dep:bigint", "dep:rsa", "alloc", "rand_core"]
+rsa-sha1 = ["rsa", "dep:sha1", "sha1/oid"]
```

Comment:
Perhaps we could just promote `sha1` to a proper feature and have this work when `rsa` and `sha1` are enabled?

Reply:
Done :v:

Reply:
I meant getting rid of this feature entirely

Reply:
My bad, I've accidentally committed to #323 instead, it's cherry picked here now